etc/profile-a-l/chatterino.profile

```diff
@@ -0,0 +1,97 @@
+# Firejail profile for Chatterino
+# Description: Chat client for https://twitch.tv
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chatterino.local
+# Persistent global definitions
+include globals.local
+
+# Also allow access to mpv/vlc, they're usable via streamlink.
+noblacklist ${HOME}/.config/mpv
+noblacklist ${HOME}/.config/pulse
+noblacklist ${HOME}/.config/vlc
+noblacklist ${HOME}/.local/share/chatterino
+noblacklist ${HOME}/.local/share/vlc
+
+# Allow Python for Streamlink integration (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+# Allow Lua for mpv (blacklisted by disable-interpreters.inc)
+include allow-lua.inc
+
+# disable-*.inc includes
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+# Also allow read-only access to mpv/VLC, they're usable via streamlink.
+mkdir ${HOME}/.config/mpv
+mkdir ${HOME}/.config/pulse
+mkdir ${HOME}/.config/vlc
+mkdir ${HOME}/.local/share/chatterino
+mkdir ${HOME}/.local/share/vlc
+# VLC preferences will fail to save with read-only set.
+whitelist ${HOME}/.local/share/chatterino
+whitelist-ro ${HOME}/.config/mpv
+whitelist-ro ${HOME}/.config/pulse
+whitelist-ro ${HOME}/.config/vlc
+whitelist-ro ${HOME}/.local/share/vlc
+# To upload images, whitelist/noblacklist their path in chatterino.local.
+#whitelist ${PICTURES}
+# For custom notification sounds, whitelist/noblacklist their path in chatterino.local.
+#whitelist ${MUSIC}
+# whitelist-*.inc includes
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+# Streamlink+VLC doesn't seem to close properly with apparmor enabled.
+#apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noprinters
+noroot
+notv
+nou2f
+# Netlink is required for streamlink integration.
+protocol unix,inet,inet6,netlink
+# Seccomp may break browser integration.
+seccomp
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+# Add more private-bin lines for browsers or video players to chatterino.local if wanted.
+private-bin chatterino,cvlc,env,ffmpeg,mpv,nvlc,pgrep,python*,qvlc,rvlc,streamlink,svlc,vlc,waf
```

Comment:
`waf` as in the build system?  Why would it be needed here?

Reply:
Assumed mpv used it since it's present in the private-bin line in mpv.profile (since #3619?), seems to only be a build/make dependency though, I'll remove it.

Reply:
I think this `waf` addition stems from https://github.com/netblue30/firejail/issues/3701. It turns up in https://github.com/netblue30/firejail/commit/f3585e53933c95d3be31bb53214145d9219ff3ea too and is still in those profiles too. I don't know if it is actually needed. Like you asked, it's indeed strange to have a build system like waf in there. In the context of this specific PR though, I assume the OP just copied what was already there to begin with. Which does seem to be the best way to proceed when not wanting to break things. We should try to determine this `waf` situation and tackle it separately if it turns out we can drop it. Just my two cents.